pkg/kubelet/kubelet.go

```diff
@@ -1245,6 +1245,14 @@ func (kl *Kubelet) StartGarbageCollection() {
 		}
 	}, ContainerGCPeriod, wait.NeverStop)
 
+	stopChan := make(chan struct{})
+	defer close(stopChan)
+	// when the high threshold is set to 100, stub the image GC manager
+	if kl.kubeletConfiguration.ImageGCHighThresholdPercent == 100 {
+		glog.V(2).Infof("ImageGCHighThresholdPercent is set 100, Disable image GC")
+		go func() { stopChan <- struct{}{} }()
```

Comment:
why not use wait.NeverStop?  i am not sure i understand this part of the change.

Reply:
wait.NeverStop is a global var, it is also used by  container GC, so i define another stopChan same as wait.NeverStop, and the stopChan  just used by imageGC to receive the stop signal 
then jump out the wait.Util() logic.